docs/user/openstack/README.md

```diff
@@ -287,6 +294,35 @@ Alternatively, if you don't control the DNS, you can add the hostnames in `/etc/
 <apps FIP> <app name>.apps.<cluster name>.<base domain>
 ```
 
+**WARNING:** *this workaround will make the API accessible only to the computer with these `/etc/hosts` entries. This is fine for your own testing (and it is enough for the installation to succeed), but it is not enough for a production deployment. In addition, if you create new OpenShift apps or routes, you will have to add their entries too.*
+
+##### External API Access
+
+If you have specified the API floating IP (either via the installer prompt or by adding the `lbFloatingIP` entry in your `install-config.yaml`) the installer will attach the Floating IP address to the `api-port` automatically.
+
+If you have created the API DNS record, you should be able access the OpenShift API.
+
+##### External Ingress (apps) Access
+
+The installer doesn't currently handle the Ingress floating IP address the same way it does the API one.
+
+To make the OpenShift Ingress access available (this includes logging into the deployed cluster), you will need to attach the Ingress floating IP to the `ingress-port` after the cluster is created.
+
+That can be done in the following steps:
+
+```sh
+openstack port show <cluster name>-<clusterID>-ingress-port
+```
+
+Then attach the FIP to it:
+
+```sh
+openstack floating ip set --port <ingress port id> <apps FIP>
+```
+
+This assumes the floating IP and corresponding `*.apps` DNS record exists.
+
+
 #### Without Floating IPs
 
 If you cannot or don't want to pre-create a floating IP address, the installation should still succeed, however the installer will fail waiting for the API.
```

Comment:
> the installation should still succeed

In my experience, it does not.

If there is no available FIP in the selected external network, the installer fails with:
```
FATAL failed to fetch Terraform Variables: failed to fetch dependency of "Terraform Variables": failed to fetch dependency of "Cluster ID": failed to fetch dependency of "Install Config": failed to fetch dependency of "Base Domain": failed to generate asset "Platform": there are no unassigned floating IP addresses available
```

Reply:
Oh crap. That's a bug, then. We intend to support deployments without floating IPs. Going to leave this as is, but log a task to have this fixed.